### ISSUE TITLE
Связанные касательные

### ISSUE THREAD

**Issue by Stas-Kholodilin** [morevnaproject/org#7]
При испозовании инструмента "Control Point Editor Tool" сделать так, чтобы можно было двигать касательные синхронно, т.е. чтобы у них длина и угол были связаны. А сейчас связаны могут быть только углы (а длина редактируется у каждой по отдельности).

**Comment by morevnaproject** [comment]
Стас, запиши, пожалуйста демо-видео как это работает в Synfig. ^__^

**Comment by morevnaproject** [comment]
Стас, напоминаю. ^__^

**Comment by Stas-Kholodilin** [comment]
https://www.youtube.com/watch?v=fvdlt5B6F_0

**Comment by morevnaproject** [comment]
@Stas-Kholodilin О, это даже больше чем я ожидал! Отлично!

@beeheemooth Может я чего-то не знаю и в OpenToonz уже есть возможность двигать каательные, как показано в видео на 0:16? - https://www.youtube.com/watch?v=fvdlt5B6F_0

**Comment by beeheemooth** [comment]
@morevnaproject Нет, в OT радиусы не связаны ни в векторах, ни в Function Curves, редактируются отдельно.

**Comment by morevnaproject** [comment]
@beeheemooth Спасибо за информацию! ^__^
